workstyle.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=workstyle autostart
+BindsTo=sway-session.target
+
+[Service]
```

Comment:
I would suggest adding
```
Environment="RUST_LOG=debug"
```
here so that systemctl status can be more helpful.

Reply:
All good :+1:

Reply:
Thanks